### PR TITLE
[JUJU-1839] Added jobs for change-user-password tests.

### PR DESCRIPTION
**Pull request by anvial**
This PR adds jobs for change-user-password tests.

### CODE REVIEW

File: jobs/ci-run/integration/gen/test-user.yml [hunk 1]
```diff
@@ -32,16 +32,128 @@
     - multijob:
         name: 'IntegrationTests-user'
         projects:
-        - name: 'test-user-lxd'
+        - name: 'test-user-test-user-login-password-lxd'
           current-parameters: true
-        - name: 'test-user-aws'
+        - name: 'test-user-test-user-login-password-aws'
```
Comment (cderici) [inline jobs/ci-run/integration/gen/test-user.yml:37]: Do we really want the duplicate wording there? `test-user-test-user-login-password-aws`

File: jobs/ci-run/integration/gen/test-user.yml [hunk 2]
```diff
@@ -32,16 +32,128 @@
...
           current-parameters: true
+        - name: 'test-user-test-user-manage-lxd'
+          current-parameters: true
+        - name: 'test-user-test-user-manage-aws'
+          current-parameters: true
+
+- job:
+    name: test-user-test-user-login-password-lxd
+    node: ephemeral-focal-8c-32g-amd64
```
Comment (cderici) [inline jobs/ci-run/integration/gen/test-user.yml:46]: Can this use `ephemeral-focal-small-amd64`?

**Review by cderici — APPROVED** [review]

File: jobs/ci-run/integration/gen/test-user.yml (hunk 1 above)
Comment (anvial) [reply jobs/ci-run/integration/gen/test-user.yml:37]: This code is auto-generated.
Template is: test-<suite-name>-test-<suite-name>-<test-name>-<cloud-name>

So it's not a part of this PR. But we can discuss changes in the template for code generation.

File: jobs/ci-run/integration/gen/test-user.yml (hunk 2 above)
Comment (anvial) [reply jobs/ci-run/integration/gen/test-user.yml:46]: Again we use the template for code generation. So, to manage these issues, we need to change the logic for example create additional config files with test-lists for "different" node types.

**Pull request merged**